packages/workspace/src/browser/quick-open-workspace.ts

```diff
@@ -81,7 +93,18 @@ export class QuickOpenWorkspace {
                 },
             });
         }
-        this.quickInputService?.showQuickPick(this.items, { placeholder: 'Type the name of the workspace you want to open' });
+        this.quickInputService?.showQuickPick(this.items, {
+            placeholder: nls.localize(
+                'theia/windowActions/openRecentPlaceholder',
+                'Type the name of the workspace you want to open'),
```

Comment:
Do you think it's worth using what vscode uses here as a placeholder (which will then be localized)?

Reply:
Good suggestion, as the string would not be localized.
Done!